renderers/src/main/java/com/pedrogomez/renderers/DefaultDiffUtilItemCallback.java

```diff
@@ -0,0 +1,17 @@
+package com.pedrogomez.renderers;
+
+import android.support.annotation.NonNull;
+import android.support.v7.util.DiffUtil;
+
+public class DefaultDiffUtilItemCallback<T> extends DiffUtil.ItemCallback<T> {
```

Comment:
The new default implementation for the items' diffing

Reply:
Great thanks!